packages/tradershub/src/components/CurrencySwitcher/CurrencySwitcher.tsx

```diff
@@ -83,7 +85,14 @@ const CurrencySwitcher = () => {
                                     <TradingAccountsList />
                                 </Modal.Content>
                                 <Modal.Footer className='grid-cols-1'>
-                                    <Button isFullWidth variant='outlined'>
+                                    <Button
+                                        isFullWidth
+                                        onClick={() => {
+                                            setIsManageAccountOpen(true);
```

Comment:
@shahzaib-deriv what is this hide for?

Reply:
<img width="512" alt="Screenshot 2024-02-27 at 3 40 31 PM" src="https://github.com/binary-com/deriv-app/assets/129842155/c05aafec-3f50-44b6-82e5-e2a5023fe475">
@mahdiyeh-deriv this `Select Account` is based on `ModalProvider` which needs to be hidden if we open the `Add or manage account` modal. For hiding we are calling `hide` from `ModalProvider`

Reply:
We are gonna remove this whole logic @mahdiyeh-deriv. Cause we gonna use Modal from deriv-com/ui

Reply:
@thisyahlen-deriv @shahzaib-deriv I see, it's really annoying to call hide first every time we want to open a different modal, I'd like to suggest to call hide on the show fucnction in the provider itself but since we want to replace it with deriv/ui, it's fine.